todo_app/trello_items.py

```diff
@@ -0,0 +1,42 @@
+from flask import trello
```

Comment:
Trello isn't a flask module unfortunately, we'll have to handle these problems ourselves with `requests`
```suggestion
```